data/categories.txt

```diff
@@ -1,4 +1,15 @@
 # Category groups referenced in many categories, should be defined before being referenced.
+# Syntax:
+# @   - string that starts with this symbol indicates the beginning of the group translations, which are used for the spotlight search.
```

Comment:
Not only for spotlight. It's also used for search of multiple types for a single word. For example, Pub or Bar are searched for when Food is entered.